packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/scanner-api/subscribable-events.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect} from 'react';
+import {
+  Navigator,
+  Screen,
+  Stack,
+  Text,
+  useScannerDataSubscription,
+  reactExtension,
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridModal = () => {
+  const {data, source} = useScannerDataSubscription();
+
+  return (
+    <Navigator>
+      <Screen name="Home" title="Home">
+        <Stack direction="horizontal" flexChildren flex={1}>
```

Comment:
Do `flexChildren` and `flex` do anything here?

Reply:
I will need to test this more tomorrow. I am having troubles deeplinking atm.